### ISSUE TITLE
Social sharing plugin

### ISSUE THREAD

**Issue by mic345** [capacitor-community/proposals#60]
## Plugin Request
Social sharing plugin which would allow users to share content

**Name**: Social Sharing
**Package**: `@capacitor-community/social-sharing`

### Platform(s)
iOS and Android

### Existing Solutions
https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin

### Description
Allow users to share content with their contacts.

**Comment by Saqib92** [comment]
Any Update??

**Comment by mic345** [comment]
https://capacitorjs.com/docs/apis/share

**Comment by Saqib92** [comment]
> https://capacitorjs.com/docs/apis/share

Yes, but I want to share specifically to Just Instagram or Facebook.

**Comment by johnborges** [comment]
@Saqib92 I don't think its possible to filter out share options using `@capacitor/share`. It defaults to show everything available.

**Comment by Saqib92** [comment]
> @Saqib92 I don't think its possible to filter out share options.

https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin

This Plugin have this Features to share specifically to other platforms.

**Comment by johnborges** [comment]
@Saqib92 There is a [feature request](https://github.com/ionic-team/capacitor-plugins/issues/69) open to add that functionality.
